src/features/game/types/craftables.ts

```diff
@@ -54,6 +55,7 @@ export type Food =
 
 export const FOODS: Record<Food, Craftable> = {
   "Pumpkin Soup": {
+    itemType: "food",
```

Comment:
Can this key just be `type`? Is there a reason that `itemType` is better? I feel like this reads better:

```
Flour: {
   type: "food"
   ...
}
```